libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1628,7 +1716,6 @@ void PbftManager::pushSyncedPbftBlocksIntoChain() {
       net->setSyncStatePeriod(period);
 
       if (executed_pbft_block_) {
-        vote_mgr_->removeVerifiedVotes();
```

Comment:
we do not need to clean those votes?

Reply:
not anymore. Purpose of this function: @brief Move all verified votes to the unverified map. Since PBFT chain has pushed new blocks, that will affect DPOS. I deleted this function